apps/spruce/src/pages/waterfall/WaterfallGrid.tsx

```diff
@@ -159,13 +161,18 @@ export const WaterfallGrid: React.FC<WaterfallGridProps> = ({
 
 const Container = styled.div``;
 
-const StickyHeader = styled(Row)`
+const StickyHeader = styled(Row)<{ atTop: boolean }>`
   position: sticky;
   top: -${size.m};
   z-index: 1;
 
   background: white;
   padding: ${size.xs} 0;
+  ${({ atTop }) =>
+    atTop
+      ? "box-shadow: unset"
+      : "box-shadow: 0 2px 2px -2px rgba(0, 0, 0, 0.5); "}
```

Comment:
[nit] I know this matches settings header shadow but I think a little extra looks good here
```suggestion
      : "box-shadow: 0 4px 4px -4px rgba(0, 0, 0, 0.5); "}
```